tests/async_type_helper_thandle_handler_ut/async_type_helper_thandle_handler_ut.c

```diff
@@ -79,7 +79,7 @@ TEST_FUNCTION(async_type_helper_thandle_copy_handler_with_NULL_dst_fails)
     ASSERT_ARE_NOT_EQUAL(int, 0, result);
 }
 
-/* Tests_SRS_ASYNC_TYPE_HELPER_THANDLE_HANDLER_01_006: [ Otherwise the copy handler shall assign src to dst by calling THANDLE_INITIALIZE. ]*/
```

Comment:
>THANDLE_INITIALIZE [](http://example.com/codeflow?start=119&length=18)

The spec in the .md should say THANDLE_INITIALIZE. And then code and tests tagging should be updated.

Reply:
Done